rules/do-you-use-version-control-with-power-bi/rule.md

```diff
@@ -40,73 +40,75 @@ In other words, the history of the changes isn't recorded anywhere.
 :::
 
 ::: good
 ![Figure: Good example - PBIP format allows recording history of changes without saving data into version control](good-example-history-recorded.png)
 :::
 
-Read the rule on [the Power BI version control features](/do-you-know-powerbi-version-control-features) to learn more about Power BI Desktop Projects, and Power BI's integration with Git.
+Historically report developers have used the .pbix or the .pbit format. These are **no longer recommended** as the new PBIP format overcomes the shortcomings of these 2 formats. 
 
 ### Editing and Committing Reports
-
-You should no longer edit or publish reports directly in the production workspace on the Power BI Service. A better process for editing and committing reports is described below.
-
-### Business Users
+### Business Users (Use Power BI Service)
 
 If you're a business user, watch the following video to get a walkthrough of the process you would follow to edit and commit reports.
 
 `youtube: https://www.youtube.com/watch?v=dlOK6QBEyQo`
-**Video: Power BI Source Control for the Business User**
+**Video: Power BI Source Control for the Business User (timebox 8:50 mins)**
 
 The entire process is done on Power BI Service (web) (except the step to create a pull request). At a high-level the steps are:
 
 1. Create a private workspace corresponding to the workspace where your report resides (1 time)
 2. Connect the private workspace to repo (1 time)
-3. Create new feature branch off ‘main’ (every time)
+3. Create new feature branch off ‘main’ (**every time**)
 4. Setup dataset connections (1 time) (take help from SysAdmins or Power BI Admins)
-5. Edit the report in Power BI Service (every time)
-6. Commit report to feature branch (every time)
-7. Create PR (pull request) to merge feature branch into ‘main’ on Azure DevOps (every time)
-8. Next time, create new feature branch on same workspace
+5. Edit the report in Power BI Service (**every time**)
+6. Commit report to feature branch (**every time**)
+7. Create PR (pull request) to merge feature branch into ‘main’ on Azure DevOps and get it reviewed by a Power BI Admin (**every time**)
+8. Get a Power BI Admin to deploy the report (**every time**)
+9. Next time, create new feature branch on same workspace
 
 If you want to update the report's data model or want more sophisticated editing features, you will need to edit the report in Power BI Desktop instead. The next section explains how you can do so.
 
-### Developers
+### Developers (Use Power BI Desktop)
 
 If you're a developer, watch the following video to get a walkthrough of the process you would follow to edit and commit reports.
 
 `youtube: https://www.youtube.com/watch?v=MpedXah-Hv0`
-**Video: Power BI Source Control for Developers**
+**Video: Power BI Source Control for Developers (timebox 5:00 mins)**
```

Comment:
I think this should just say (5 mins) after the video name
as per the rule https://www.ssw.com.au/rules/right-format-to-write-videos-time-length/

Reply:
Done